components/StatisticsCard.vue

```diff
@@ -27,12 +45,12 @@ export default {
   font-size: 12px;
```

Comment:
```suggestion
  align-self: flex-start;
  font-size: 12px;
```

Пока все элементы контейнера жмутся ко дну, text будет липнуть к началу.